testscript/testscript.go

```diff
@@ -126,8 +126,10 @@ func Run(t *testing.T, p Params) {
 type T interface {
 	Skip(...interface{})
 	Fatal(...interface{})
+	Fatalf(string, ...interface{})
 	Parallel()
 	Log(...interface{})
+	Logf(string, ...interface{})
```

Comment:
Technically adding methods here is a backwardly incompatible change, and it's not very necessary, so I think it's probably worth just using `Log` and `Fatal` with `fmt.Sprintf` instead, as we're already doing in at least one place for Fatal.

Reply:
Good point. Done